README.md

```diff
@@ -121,60 +91,51 @@ use all your colors in the components which have a `data-color` prop:
     // ...other compilerOptions
     "types": [
       // ...other types
-      "@digdir/designsystemet-theme"
+      "@digdir/designsystemet-theme" or "<custom-theme>/colors.d.ts",
+      "@digdir/designsystemet-react/react-types",
       ]
   },
 }
 ```
 
-#### When using a custom theme
-The CLI `designsystemet tokens build` command will output a `colors.d.ts` file to your chosen output directory.
-In the example, replace `<your-path>` with the actual path you used when running the command.
+### 2. Font
 
-```jsonc
-{
-  // ...other settings
-  "compilerOptions": {
-    // ...other compilerOptions
-    "types": [
-      // ...other types
-      "<your-path>/colors.d.ts"
-      ]
-  },
-}
-```
+You are free to use any font-family.
 
-### 5. Add editor hints for data-color & data-size on HTML elements (optional)
+The components are designed and developed using the [Inter font](https://github.com/rsms/inter) so variations might occur if a different font is used.
 
-You may want editor hints for `data-color` and `data-size` attributes on HTML elements
-like `<span>` or `<div>`, since these attributes can affect components nested within
-these elements.
+#### 2.1 Use Inter font (optional)
 
-This requires augmenting React's built-in types, and is therefore opt-in. If you want this,
-add the following to your `tsconfig.json`:
+Add the `<link>` tag in `<head>`, and set `font-family` to `Inter` in your global css file.
 
-```jsonc
-{
-  // ...other settings
-  "compilerOptions": {
-    // ...other compilerOptions
-    "types": [
-      // ...other types
-      "@digdir/designsystemet-react/react-types"
-      ]
-  },
-}
+##### HTML
+
+```html
+<link
+  rel="stylesheet"
+  href="https://altinncdn.no/fonts/inter/v4/inter.css"
+/>
 ```
 
+##### CSS
+
+```css
+body {
+  font-family: 'Inter', sans-serif;
+  font-feature-settings: 'cv05' 1; /* Enable lowercase l with tail */
+}
+```
 
+The `font-feature-settings` adds a tail to lowercase `L`'s.
+If you choose to install the font in a different way, remember to include the `400`, `500` and `600` font weights.
```

Comment:
```suggestion
If you chose to install the font in a different way, remember to include the `400`, `500` and `600` font weights.
```

Reply:
We are talking about present-tense. Chose is past-tense

Reply:
Denne er forsåvidt rett begge vega - men når eg las teksten over, og så denne - så virka "chose" (valgte) bedre enn "choose" (velger)

Reply:
We are reminding them about something they need to do in the future when they add a font. "choose" is the correct tense.

Reply:
```suggestion
If you install the font in a different way, remember to include the `400`, `500` and `600` font weights.
```